modules/transformations/redshift/doc/ST_GREATCIRCLE.md

```diff
@@ -0,0 +1,24 @@
+### ST_GREATCIRCLE
+
+{{% bannerNote type="code" %}}
+transformations.ST_GREATCIRCLE(start_point, end_point, n_points)
+{{%/ bannerNote %}}
+
+**Description**
+
+Calculate great circles routes as LineString.
+
+* `start_point`: `GEOMETRY` source point feature.
+* `end_point`: `GEOMETRY` destination point feature.
+* `n_points` (optional): `INT` number of points. If `NULL` the default value `100` is used.
```

Comment:
Same overloading situation that in `ST_DESTINATION`. You can add the other version + example + update test.